main/modes/music/jukebox/jukebox.c

```diff
@@ -629,7 +593,7 @@ jukeboxSong_t sfx_ultimatettt[] = {
         .name     = "Marker",
     },
     {
-        .filename = "`.mid",
+        .filename = "uttt_win_s.mid",
```

Comment:
Tsk tsk tsk

Reply:
I have no idea how or when this and the pong typo got into the source, especially since the pong stuff should have been there from last year. Guess I'll go on a git blame marathon later to see when I introduced these. Either way, thanks for fixing them!